### ISSUE TITLE
about classification

### ISSUE THREAD

**Issue by WhiteHe001** [zhangtaolab/Plant_DNA_LLMs#6]
outs['logits'].shape is torch.Size([2, 512, 8005])，outs['hidden_states'][-1].shape is torch.Size([2, 512, 768]) ，Is 512 mean seq_len?Is  8005 mea emb_dim?  Which one should I use for classification

**Comment by ljq12697** [comment]
512 mean the max_token_length. 8005 mean the vocab_size, which is also known as num_embeddings. embd_dim is the 'n_embd' parameter in transformer config.json.
For classification, you should use the n_embd (here is 768) as input and num_labels as output.

**Comment by WhiteHe001** [comment]
thanks
